tests/gordo_components/dataset/test_dataset.py

```diff
@@ -194,6 +225,46 @@ def test_row_filter(self):
 
         self.assertEqual(3, len(X))
 
+    def test_aggregation_methods(self):
+        """Tests that it works to set aggregation methods"""
+
+        tag_list = [
+            SensorTag("Tag 1", None),
+            SensorTag("Tag 2", None),
+            SensorTag("Tag 3", None),
+        ]
+        start = dateutil.parser.isoparse("2017-12-25 06:00:00Z")
+        end = dateutil.parser.isoparse("2017-12-29 06:00:00Z")
+
+        # Default aggregation gives no extra columns
+        X, _ = TimeSeriesDataset(
+            MockDataSource(), start, end, tag_list=tag_list
+        ).get_data()
+
+        self.assertEqual((577, 3), X.shape)
+
+        # Using two aggregation methods give 2*tags columns
+        X, _ = TimeSeriesDataset(
+            MockDataSource(),
+            start,
+            end,
+            tag_list=tag_list,
+            aggregation_methods=["mean", "max"],
+        ).get_data()
+
+        self.assertEqual((577, 6), X.shape)
+        self.assertEqual(
+            list(X.columns),
+            [
+                "Tag 1_mean",
+                "Tag 1_max",
+                "Tag 2_mean",
+                "Tag 2_max",
+                "Tag 3_mean",
+                "Tag 3_max",
+            ],
```

Comment:
Does this last part make the first test you wrote redundant?

Reply:
Hmm seems like it. Did I maybe forget that I wrote a test and then basicly rewrite it? IDK, but I will delete the first.  Goodie catchie